data/hai/hai reveal 1 intro.txt

```diff
@@ -400,8 +400,8 @@ mission "Hai Tour [2] Alta Hai"
 		has "ZZ Hai Tour [1] Stonebreak: done"
 	on offer
 		conversation
-			`This star system has an unusual number of metallic asteroids, which you noticed the Hai harvesting for the raw materials before you landed. Stonebreak itself is home to incredibly large cities: massive, clearly planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards.`
-			`	From above, this city has almost perfect radial symmetry, a sure sign that it was planned and built as a single unit rather than expanding at random over time. Every single building in the city is a dull and slightly irregular gray color, all made from the same material which seems to be some metallic alloy, and the corners of each building are slightly beveled instead of meeting at right angles.`
+			`This star system has an unusual number of metallic asteroids, and you noticed the Hai harvesting for the raw materials before you landed. Stonebreak itself is home to incredibly large cities: massive, clearly planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards.`
```

Comment:
```suggestion
			`This star system has an unusual number of metallic asteroids, which you noticed the Hai harvesting for the raw materials before you landed. Stonebreak itself is home to incredibly large cities: massive, clearly planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards.`
```
I'm sure this was correct before. What the Hai are harvesting could be less clear now with this change. Otherwise:
```suggestion
			`This star system has an unusual number of metallic asteroids, and you noticed the Hai harvesting them for the raw materials before you landed. Stonebreak itself is home to incredibly large cities: massive, clearly planned communities built in concentric rings, where housing space and community buildings alternate with zones of factories and shipyards.`
```

Reply:
Yea, the original was fine.